applications/ecallmgr/src/ecallmgr_call_command.erl

```diff
@@ -110,6 +110,13 @@ get_fs_app(_Node, _UUID, JObj, <<"hangup">>) ->
         'true' ->  {<<"unbridge">>, <<>>}
     end;
 
+get_fs_app(_Node, UUID, JObj, <<"audio_level">>) ->
+    Action = wh_json:get_value(<<"Action">>, JObj),
+    Level = wh_json:get_value(<<"Level">>, JObj),
+    Mode = wh_json:get_value(<<"Mode">>, JObj),
+    Data = <<UUID/binary, " ", Action/binary, " ", Mode/binary, " mute ", Level/binary>>,
```

Comment:
Well, I thought this at first too, but after testing this is the only reliable way to mute and unmute. If you use level it switches to another mode which messes up things and using start/stop is more complicated with same result. I have this working in my training VM with FS 1.7 but it also works identically in F 1.4 if you want to test it.

I have this metaflow as example for mute and unmute which is the primary use-case. With more testing other configs could work too but may also need fixes in freeswitch.

      "4": {
        "module": "audio_level",
        "data": {
          "action": "start",
          "mode": "read",
          "level": "-4"
        }
      },
      "5": {
        "module": "audio_level",
        "data": {
          "action": "start",
          "mode": "read",
          "level": "0"
        }
      }
    },

It produces for mute ```uuid_audio(Zj3a1bneFKXwhhNnfwTzRypWNz0MAOLS start read mute -4)``` and ```uuid_audio(Zj3a1bneFKXwhhNnfwTzRypWNz0MAOLS start read mute 0)``` for unmute. Do you have a batter idea to do mute/unmute?